config/plugins.js

```diff
@@ -1,17 +1,9 @@
 const { resolve } = require('path');
+const pckg = import('@openshift/dynamic-plugin-sdk-webpack');
```

Comment:
Why Async?

Reply:
The webpack config uses commonJS (we can move it to esmodule if we want to, but that is probably outside of scope of this PR) and this plugin is ES module so we have to use `import` as a fallback

Reply:
Ah right -- that's unfortunate. Okay, thanks for the info.